operators/pkg/utils/test/integration.go

```diff
@@ -26,6 +34,11 @@ var log = logf.Log.WithName("integration-test")
 
 // RunWithK8s starts a local Kubernetes server and runs tests in m.
 func RunWithK8s(m *testing.M, crdPath string) {
+	if err := apis.AddToScheme(scheme.Scheme); err != nil {
+		fmt.Println("fail to add scheme")
+		panic(err)
```

Comment:
t.Fatalf("%s: %s", msg, err.Error()) could be a cleaner way to handle this.

Reply:
well, I don't have a `t` here?

Reply:
Heck. Let's keep it that way.